docs/4.examples/handle-session.md

```
@@ -148,6 +148,45 @@ Every option is optional except `password`. The `name` option is worth calling o
+Resealing is the expensive part of a session, so H3 does not do it on every request: it reseals only once more than half the window has been used, and updating the session counts as a reseal. An active user therefore never gets signed out, but the recorded last-seen time can trail the real one by up to half the window:
+
+```js
+// idleTimeout: 60 * 30
+// Sign-out happens 15 to 30 minutes after the last request, never later.
+```
+
+Halve `idleTimeout` if you need the shorter end of that range to be your real limit.
+
+> [!NOTE]
+> Only cookie sessions slide. A session sent through the `x-{name}-session` header cannot be resealed, so it expires `idleTimeout` after its seal was issued.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)